src/Web/Internal/FormUrlEncoded.hs

```diff
@@ -267,6 +267,10 @@ class ToForm a where
 
 instance ToForm Form where toForm = id
 
+instance ToForm Void where toForm _ = Form HashMap.empty
```

Comment:
absurd

Reply:
You are right. Like this?

https://github.com/Daniel-Diaz/http-api-data/commit/b7e4ce9b33b19ccf9fa00a9693461a5dd170788c